ui/app/components/token-view-balance/token-view-balance.component.js

```diff
@@ -0,0 +1,94 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import Button from '../button'
+import Identicon from '../identicon'
+import TokenBalance from '../token-balance'
+import CurrencyDisplay from '../currency-display'
+import { SEND_ROUTE } from '../../routes'
+import { ETH } from '../../constants/common'
+
+export default class TokenViewBalance extends PureComponent {
```

Comment:
Would the `TokenViewBalance` and `TokenView` components more accurately be called `TransactionViewBalance` and `TransactionView`?